refactor(pcc.user): refactor the section user for pcc federation

ghyenne opened the pull request:
<!--
Hello 👋 Thank you for submitting a Pull Request.

Have any questions? Check out the contributing docs at https://github.com/ovh/manager/blob/master/CONTRIBUTING.md

-->

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-10026` <!-- target branch -->
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-10026 <!-- prefix each issue number with "Fix #", if any -->
| License          | BSD 3-Clause

<!--
  Before submitting your PR, please review the following checklist:
-->

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
I refactor the section user for pcc federation
<!-- Write a brief description of the changes introduced by this PR -->

## Related

<!-- Link dependencies of this PR -->

anooparveti commented on packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/delete/translations/Messages_fr_FR.json:
```diff
@@ -0,0 +1,6 @@
+{
+  "dedicatedCloud_USER_delete_title": "Supprimer un utilisateur",
+  "dedicatedCloud_USER_delete_message": "Etes-vous certain(e) de vouloir supprimer l'utilisateur <b>{{t0}}</b>?",
```
**nitpick:** remove html from TRANS files

ghyenne replied:
It's applied.

anooparveti commented on packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/delete/translations/Messages_fr_FR.json:
```diff
@@ -0,0 +1,6 @@
+{
+  "dedicatedCloud_USER_delete_title": "Supprimer un utilisateur",
+  "dedicatedCloud_USER_delete_message": "Etes-vous certain(e) de vouloir supprimer l'utilisateur <b>{{t0}}</b>?",
+  "dedicatedCloud_USER_delete_success": "La demande de suppression de l'utilisateur <b>{{t0}}</b> a bien été prise en compte",
```
**nitpick:** remove html from TRANS files

ghyenne replied:
It's applied.

anooparveti commented on packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/disable/translations/Messages_fr_CA.json:
```diff
@@ -0,0 +1,6 @@
+{
+  "dedicatedCloud_USER_disable_title": "Désactiver un utilisateur",
+  "dedicatedCloud_USER_disable_message": "Etes-vous certain(e) de vouloir désactiver l'utilisateur <strong>{{t0}}</strong> ?",
```
**nitpick:** remove html from TRANS files

anooparveti reviewed (commented):
Could you please have a look at the Security Hotspots reported by SonarCloud?

ghyenne commented:
> Could you please have a look at the Security Hotspots reported by SonarCloud?

I corrected the Smells code, but for the security Hotspots I think that sonarcloud is based on the variable name password when it is not a password but a route to access the page (this is not a part of the code that I modified, it was already like that previously).

sachinrameshn commented on packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/enable/translations/Messages_fr_FR.json:
```diff
@@ -0,0 +1,7 @@
+{
+  "dedicatedCloud_USER_enable_title": "Activer un utilisateur",
+  "dedicatedCloud_USER_enable_infos": "Cet utilisateur est un administrateur temporaire de votre infrastructure. Il sera désactivé dans 2h.",
+  "dedicatedCloud_USER_enable_message": "Etes-vous certain(e) de vouloir activer l'utilisateur <strong>{{t0}}</strong> ?",
```
Suggestions: remove html from TRANS files

ghyenne replied:
It's applied.

ghyenne commented on packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/password/reset/translations/Messages_fr_FR.json:
```diff
@@ -0,0 +1,20 @@
+{
+  "dedicatedCloud_USER_change_password": "Changer le mot de passe",
+  "dedicatedCloud_users_password_reset_not_allowed": "Pour changer votre mot de passe, veuillez utiliser votre interface web certifiée en vous rendant à l'adresse :",
+  "dedicatedCloud_USER_change_password_confirmation": "Indiquez le nouveau mot de passe pour l'utilisateur <b>{{t0}}</b> :",
```
It's applied.
